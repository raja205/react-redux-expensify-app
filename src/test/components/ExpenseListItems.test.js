import React from 'react';
import { Link } from 'react-router-dom';
import {shallow} from 'enzyme'
import ExpenseListItem from '../../components/ExpenseListItem'
import expenses from '../fixtures/expenses'

test('should render expense list items correctly',()=>{
    const wrapper = shallow(<ExpenseListItem {...expenses[0]}/>)
    expect(wrapper).toMatchSnapshot()
})
