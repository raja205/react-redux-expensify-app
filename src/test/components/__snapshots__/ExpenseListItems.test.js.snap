// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list items correctly 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    1.9500
    -
    January 1st, 1970
  </p>
</div>
`;
